packages/fields-auto-increment/src/test-fixtures.js

```diff
@@ -12,7 +12,7 @@ export const skipCreateTest = true;
 export const skipUpdateTest = true;
 
 // `AutoIncrement` field type is not supported by `mongoose`. So, we need to filter it out while performing `API` tests.
-export const unSupportedAdapterList = ['mongoose'];
+export const unSupportedAdapterList = ['mongoose', 'prisma'];
```

Comment:
Why is Prisma here? Isn't AutoIncrement the default primary key for Prisma?

Reply:
Oh, mostly I had this disabled because tests weren't passing for reasons I didn't have a chance to look into. Let me put this to the top of my fields matrix work and I should be able to get this field type with all green ticks now 👍

Reply:
Ah yes, now I recall, these failures are related to the `Number` stuff throughout the adapter code. Need to fix both of those together.

Reply:
Fixed.